### ISSUE TITLE
Backups Start as soon as User Logs in

### ISSUE THREAD

**Issue by katmoody** [humanmade/backupwordpress#1061]
I think this is partially related to #1036 - basically every schedule sometimes will attempt to run, but not every time. 

I've been checking this on sites I visit - I think it's the cron kicking in because it tends to happen on low traffic sites. My thinking is that all the schedules are waiting for a site visit to kick off so they all go at the same time when someone logs in to check out their site ... possible? 

I'm listing it as a bug, because I think it's actually brought a site or two down by killing their server processes and overloading them but not sure it's really a bug?

**Comment by pdewouters** [comment]
Not a bug, that's how WP Cron works.

**Comment by katmoody** [comment]
I understand that (well, mostly) - my concern is that it is causing the schedules to kick off all at once - shouldn't it at least be limited to one schedule at a time?

**Comment by pdewouters** [comment]
Yes, I thought there was an issue open for that. but can't find it. Several schedule running in parallel is not good. But that would require some sort of queueing system
